src/Microsoft.SqlTools.ServiceLayer/Connection/ListDatabaseRequestHandler.cs

```diff
@@ -199,28 +188,6 @@ protected override void SetProperties(DbDataReader reader, DatabaseInfo database
     /// </summary>
     class SqlDBDatabaseDetailHandler : BaseDatabaseDetailHandler
```

Comment:
This class is pointless now - I'd just get rid of it and make the base class a regular class instead of abstract.